### DISCUSSION TITLE
SOLVED: OpenEVSE Integration does not show up

### DISCUSSION THREAD

**Discussion by fhteagle** [firstof9/openevse#119] [General]
Just wanted to leave a problem / solution that had me scratching my head until the coffee kicked in. 

I originally tried to install this integration per the readme.md instructions here, except that I used a symlink to the openevse folder where I keep most of my cloned git repository files. Permissions were not an issue on any of the files/folders. Only the other older / useless "OpenEVSE" integration would show up using this method.

Removing the symlink and actually copying the files instead meant that everything worked as expected. I guess HomeAssistant in a docker container is prevented from following the symlink...?

**Comment by firstof9** [comment]
I think it cannot follow symlinks unless you give it privileged access, it's a security thing AFAIK inherent in docker.
